stp_zmq/zstack.py

```diff
@@ -914,6 +928,10 @@ def clearLocalKeep(self):
     def clearRemoteKeeps(self):
         pass
 
+    def checkMsgLenWithException(self, msg: bytes):
```

Comment:
Probably it makes sense to create a special Validator component for messages on the transport level (`TransportValidator`) and put this check there.
We may have other checks on the transport level in future.